src/components/Messages/PreviewList.tsx

```diff
@@ -125,16 +103,16 @@ const PreviewList: FC<Props> = ({ className, selectedConversationKey }) => {
             )}
           >
             Requested
-            <span className="text-sm font-bold ml-2 bg-gray-200 px-3 py-0.5 rounded-2xl">
-              {Array.from(requestedProfiles.values()).length > 99
-                ? '99+'
-                : Array.from(requestedProfiles.values()).length}
-            </span>
+            {requestedCount > 0 && (
+              <span className="text-sm font-bold ml-2 bg-gray-200 px-3 py-0.5 rounded-2xl">
+                {requestedCount > 99 ? '99+' : requestedCount}
+              </span>
+            )}
           </div>
         </div>
         {selectedTab === 'Requested' ? (
           <div className="p-2 mt-1 text-sm bg-yellow-100 text-yellow-800">
-            These conversations are from Lenster profiles that you don't currently follow.
+            These conversations are from Lens profiles that you don't currently follow.
```

Comment:
+ @darickdang @bigint I tweaked the copy since profiles aren't a Lenster-specific thing. Lmk if you want me to revert!